resources/views/components/game/link-buttons/index.blade.php

```diff
@@ -47,7 +47,7 @@
 
 <ul class="flex @if ($variant === 'stacked') flex-col @endif gap-2">
     @if (in_array('forum-topic', $allowedLinks))
-        @if ($game->ForumTopicID)
+        @if ($game->ForumTopicID && ForumTopic::find($game->ForumTopicID)?->exists())
```

Comment:
I feel like this is an unnecessary bandage. There are other references to the ForumTopicID that would be similarly effected. For example, the claim button changes to "make claim and forum topic" if the value is null.

We should probably add the foreign key and then run the query to purge bad data.

Reply:
Sounds good. I'll close this and #2357. I'll then open a PR to add the foreign key.